### ISSUE TITLE
Stage 3: Wires frome IE cant damage you

### ISSUE THREAD

**Issue by Manteso** [DarkPacks/SevTech-Ages#3875]
## Bug Report

Uninsulated wires from Immersive engineering cant damage players or mobs.
This happens every time after server start. Reconecting wires fix them.
## Expected Behaviour

HV wires should damage mobs.
## Possible Solution

Worked properly on previous version (3.1.1)
## Steps to Reproduce (for bugs)

1. Connect HV wires to generator and machine
2. Restart server

## Logs

* Client/Server Log:
* Crash Log: no crash

## Client Information

* Modpack Version: 3.1.2
* Java Version: jre 1.8.0_221
* Launcher Used: Twitch
* Memory Allocated: 8 GB
* Server/LAN/Single Player: Server
* Optifine Installed: Yes
* Shaders Enabled: No

## World Information

* Modpack Version world created in: 3.1.2

* Additional Content Installed: No additions

## Server Information
* Java Version: jre 1.8.0_221
* Operating System: Windows 10 (1903)
* Hoster/Hosting Solution: Private home server
* Sponge (Non-Vanilla Forge) Server:

**Comment by sk2048** [comment]
Immersive Engineering has not been updated since 3.1.1

A wire will only cause shocks if flux is actively flowing between the two ends. Breaking and replacing will cause the internal buffer to fill on the connectors/relays and so will allow the wires to electrocute for a period after breaking/replacing. Are you certain this isn’t what is happening?

**Comment by Manteso** [comment]
Yes, it is not what happening. Wire between charged capacitor and working pumpjack stops damage you after server restart. Pump doesnt stop working after that, capacitors charge decreasing as should be. That means flux still flow through wire. breaking only wire and connecting it again fix this behaviour only for that particular wire.

**Comment by sk2048** [comment]
I don't know why this behaviour would have changed between 3.1.1 and 3.1.2, but I will test it next time I have my server up and get back to you.

**Comment by Manteso** [comment]
Today i tested this in single player. Issue happens the same way, you make capacitor and power it up then logout. After login you can see how capacitor is still charging but wire will not damage you.
